modules/ui/combobox.js

```diff
@@ -390,10 +390,37 @@ export function uiCombobox(context, klass) {
                 })
                 .attr('title', function(d) { return d.title; })
                 .each(function(d) {
+                    var option = d3_select(this);
                     if (d.display) {
-                        d.display(d3_select(this));
+                        d.display(option);
                     } else {
-                        d3_select(this).text(d.value);
+                        option.text(d.value);
+                    }
+                    // Only for comment combobox: add cross button, floated right, minimal style
+                    if (klass === 'comment') {
+                        option.append('button')
+                            .attr('class', 'combobox-delete')
+                            .attr('title', 'Delete this comment')
```

Comment:
User facing texts should never be hardcoded to English. See https://github.com/openstreetmap/iD/wiki/How-to-Use-iD%27s-Localizer-Module for how to add and use translatable strings.